config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json

```diff
@@ -27,11 +27,20 @@
   },
   {
     "description": "Ridge CSI version",
-    "displayname": "CSI version",
-    "name": "csi-version",
+    "displayname": "Ridge CSI version",
+    "name": "ridge-csi-version",
     "type": "string",
     "required": "true",
     "category": "config",
     "mutable": "true"
+  },
+  {
+    "description": "Ridge CSI URL",
+    "displayname": "Ridge CSI URL",
+    "name": "ridge-csi-url",
+    "default": "https://tech.ridge.co",
+    "type": "string",
+    "required": "true",
```

Comment:
`"required": "true"` ---> `"required": "false"` since we have added default value

Reply:
Sorry I forgot to remove the default value. So I removed it and kept the `required`. Is it ok?

Reply:
Looks good now